docs/src/main/paradox/consumer.md

```diff
@@ -186,3 +186,15 @@ Scala
 
 Java
 : @@ snip [consumerMetrics](../../test/java/sample/javadsl/ConsumerExample.java) { #consumerMetrics }
+
+
+## Listening for rebalance events
+
+You may set up an rebalance event listener actor that will be notified when your consumer will be assigned or revoked 
```

Comment:
two callbacks?
mention the message types instead

Reply:
yeap was fixing; fixed